docs/src/development/local/ddev.md

```diff
@@ -0,0 +1,204 @@
+---
+title: Use DDEV for local development
+sidebarTitle: DDEV
+weight: 1
+---
+
+[DDEV](https://ddev.readthedocs.io/en/stable/) is an open-source tool to get local PHP development environments.
+It allows you use Docker in your workflows without needing 
+
+
+a Docker-based local development tool that plays nicely with Platform.sh.
+It allows you to have fully containerized environments to run everything locally
+without having to install tools (such as the Platform.sh CLI) on your machine.
+It's maintained by a community of developers and is a viable option for most Platform.sh projects.
+
+This guide assumes you have a project already running within Platform.sh.
+If you're starting from scratch, first [create a project from a template](https://console.platform.sh/org/create-project/template).
+
+## Before you begin
+
+Make sure your computer meets the [system requirements for Docksal](https://docs.docksal.io/getting-started/setup/#system-requirements).
```

Comment:
:lady_beetle: This page is about DDEV, why are we referring to Docksal and Docksal commands throughout the document?
Was this file added by mistake? I don't see clearly the link between the scope of the PR and this file.

Reply:
Yes, the file was added by mistake.